frontend/packages/dev-console/src/plugin.tsx

```diff
@@ -465,6 +465,34 @@ const plugin: Plugin<ConsumedExtensions> = [
         ).PipelinesPage,
     },
   },
+  {
+    type: 'Page/Route',
+    properties: {
+      perspective: 'dev',
+      exact: true,
+      path: [`/k8s/ns/:ns/${referenceForModel(PipelineModel)}/~new/builder`],
+      loader: async () =>
+        (
+          await import(
+            './components/pipelines/pipeline-builder/PipelineBuilderPage' /* webpackChunkName: "pipeline-builder-page" */
+          )
+        ).default,
+    },
+  },
+  {
+    type: 'Page/Route',
+    properties: {
+      perspective: 'dev',
+      exact: true,
+      path: [`/k8s/ns/:ns/${referenceForModel(PipelineModel)}/~new/:pipelineName/builder`],
```

Comment:
Why `~new` in the edit url?
`/k8s/ns/:ns/${referenceForModel(PipelineModel)}/:pipelineName/builder`

Reply:
I did that originally... it's caught by the Details page and thinks I want a tab in there. I am unsure how to override that. Order in the Plugin file doesn't seem to do it.

Reply:
Apparently I was wrong... not sure how I tested that and didn't get it to work. 🤷‍♂ Fixed now.